### PR TITLE
Added support for AAAA records by default

### PR DESCRIPTION
**Pull request by thekevinbrown**
Also added optional override to disable them if someone wants to.

### CODE REVIEW

File: packages/cdk-static-website/src/website-alias-record.ts [hunk 1]
```diff
@@ -16,6 +17,9 @@ export interface WebsiteAliasRecordProps {
 
   /** Target for the alias record */
   readonly target: IAliasRecordTarget;
+
+  /** We support IPv6 and add an AAAA record by default, but you can turn it off */
+  readonly disableIPv6?: boolean;
```
Comment (hupe1980) [inline packages/cdk-static-website/src/website-alias-record.ts:22]: Why didn't you implement it as an opt-in feature?

Comment (thekevinbrown) [reply packages/cdk-static-website/src/website-alias-record.ts:22]: Because IPv6 has been around for 8 years, and increasingly our services should "Just Work" with it. I expect that the normal case for most people would be to want IPv6, and that if someone were to disable it, that'd be because they have a specific operational requirement or other exception that temporarily prevents them from using IPv6.

TL;DR: It's 2020 and I feel like IPv6 support should be the default, not something you opt in to. That's a subjective opinion, but that's why I implemented it this way.

**Comment by thekevinbrown** [comment]
@hupe1980, thanks for the review, I believe I've actioned the feedback. Let me know if there are any other changes needed.

File: packages/cdk-static-website/src/website-alias-record.ts (hunk 1 above)
Comment (hupe1980) [reply packages/cdk-static-website/src/website-alias-record.ts:22]: Alright thx
